frontend/apps/marketing/src/themes/csforall/constants/colors.ts

```diff
@@ -3,6 +3,7 @@ export const COLORS = {
   white: '#FFF',
   brandPrimary: '#2C079F',
   brandSecondary: '#CA01E4',
+  brandPurpleStrong: '#9A00A9',
```

Comment:
You can use the MUI auto generated `theme.palette.secondary.dark` color instead of this since it's just for a hover. It's what I'm using on the Buttons. 

If you feel strongly about keeping this as-is please update the Button hover color and rename this to `brandSecondaryDark`. I'd like to stay away from color-specific naming patterns.